metpy/calc/tests/test_thermo.py

```diff
@@ -123,6 +123,17 @@ def test_basic(self):
         assert_almost_equal(l, 864.761 * units.mbar, 2)
 
 
+class TestLFC(object):
+    def test_basic(self):
```

Comment:
I don't know how I didn't catch this earlier. Instead of using a class, can you just make this a test function, like `test_lfc_basic()`?